hxmath/math/Vector2.hx

```diff
@@ -839,5 +839,20 @@ abstract Vector2(Vector2Type) from Vector2Type to Vector2Type
         return self.clone()
             .rotateRight();
     }
+
+    private inline function set_length(v:Float):Float
+    {
+        var self:Vector2 = this;
+        self.normalizeTo(v);
+        return v;
+    }
+
+    private inline function set_angle(v:Float):Float
```

Comment:
You can call this parameter "angle." Haxe will resolve it properly in the function body.

Reply:
Will do.

Reply:
Done.